### ISSUE TITLE
Missing Starting Natural Armor Bonus

### ISSUE THREAD

**Issue by KrispyXIV** [Balkoth-dev/WOTR_MAKING_FRIENDS#2]
The +2 Natural Armor bonus from the Eidolon _Base Form_ is missing.  Tested Biped, Abberant, Serpentine.

**Comment by KrispyXIV** [comment]
Actually, Improved Natural Armor Evolution doesn't work either?  Could this be some sort of stacking issue with other sources of natural armor? - TTT issue, turned off TTT and stacking works as expected.

**Comment by KrispyXIV** [comment]
All good.

**Comment by Balkoth-dev** [comment]
Yes, TTT causes some stacking issues because it doesn't consider the same source when checking stacking. I'll probably fix this at some point by making an option to change the type for TTT users.

**Comment by YLMstring** [comment]
Or be as lazy as me
Simply make it an armor focus bonus
